packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -399,3 +400,22 @@ pool.addClient('app-client', {
   }
 });
 ```
+
+### Domains
+
+After setting up an [app client](#app-clients), the address for the user pool's sign-up and sign-in webpages can be
+configured using domains. There are two ways to set up a domain - either the Amazon Cognito hosted domain can be chosen
+with an available domain prefix, or a custom domain name can be chosen. The custom domain must be one that is already
+owned, and whose certificate is registered in AWS Certificate Manager.
+
+The following code sets up a user pool domain in Amazon Cognito hosted domain with the prefix 'my-awesome-app' -
+
+```ts
+const pool = new UserPool(this, 'Pool');
+pool.addDomain('domain', {
+  cognitoDomainPrefix: 'my-awesome-app',
+});
+```
+
+The `UserPoolDomain` construct exposes a `cloudFrontDomainName` attribute that returns the CloudFront domain name. This
```

Comment:
Does it make sense to call this `cloudFrontDomainName` if it can also be used as a Route53 target?

Reply:
> Does it make sense to call this cloudFrontDomainName if it can also be used as a Route53 target?

It's the CF domain name. What else can we call it?

> Can I ask that you add a corresponding class to aws-route53-targets at the same time?

Was planning to do it as a separate PR, but sure, I'll add it here.

Reply:
Unfortunately, this will have to wait for https://github.com/aws/aws-cdk/pull/7361. 

How strongly do you feel about this? If you do, I'm ok with waiting to merge that before updating this.

Reply:
I don't feel super strongly, I'm just worried we'll forget. But I guess it's not that important.